.openapi-generator/FILES

```diff
@@ -373,10 +381,10 @@ lib/mux_ruby/models/web_input.rb
 lib/mux_ruby/models/web_input_response.rb
 lib/mux_ruby/version.rb
 mux_ruby.gemspec
+spec/api/drm_configurations_api_spec.rb
 spec/api_client_spec.rb
 spec/configuration_spec.rb
-spec/models/update_live_stream_new_asset_settings_spec.rb
-spec/models/update_user_agent_restriction_request_spec.rb
-spec/models/user_agent_restriction_request_spec.rb
-spec/models/user_agent_restriction_settings_spec.rb
+spec/models/drm_configuration_response_spec.rb
```

Comment:
@philcluff this doesn't seem right... not sure why update live stream and user agent restrictions... this makes me think we missed a cherry-pick

Reply:
the files still exist...

Reply:
yeah I'm at a loss here, I don't know what's happening

Reply:
This `FILES` list seems to get out of sync fairly often, I'm unclear if it's actually used for anything.